File: Hunt_Career_Playwright/pages/SavePage.js
import { BasePage } from "./BasePage";

export class SavePage extends BasePage {
  constructor(page) {
    super(page);
    this.page = page;
  }
    savedJobCard(jobTitle) {
      return this.page.locator(`//h3[contains(text(), '${jobTitle}')]`);
    }
}

File: Hunt_Career_TestNG_Hybrid_Framework/src/main/java/com/huntcareer/qa/pages/SavePage.java
package com.huntcareer.qa.pages;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class SavePage extends PageBase {

    public SavePage(WebDriver driver) {
        super(driver);
    }

    public WebElement getSavedJobCard(String jobTitle) {
        return driver.findElement(By.xpath("//h3[contains(text(), '" + jobTitle + "')]"));
    }
}